### PR TITLE
restructure the repo for i18n

### PR DESCRIPTION
**Pull request by kjhenner**

### CODE REVIEW

File: ja_jp/assets [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/Henner/Source/puppet-quest-guide/assets
```
Comment (greglarkin) [inline ja_jp/assets]: Is your personal path on your machine supposed to be embedded in these files?

Comment (kjhenner) [inline ja_jp/assets]: Oh yeah, good catch. Not entirely sure how git/GitHub handles symlinks, but I guess these should be relative rather than absolute?

**Comment by greglarkin** [comment]
Does this file (https://github.com/puppetlabs/puppet-quest-guide/blob/master/version.md) need to be localized? It's included by all of the quest markdown files.

**Comment by kjhenner** [comment]
@greglarkin No--it's contents are just "Quest Guide v2.0.0" and "Quest Guide", as the name of the product, won't be translated.

**Pull request merged**